src/main/java/uk/gov/companieshouse/authcodenotification/service/PrivateDataRetrievalService.java

```diff
@@ -16,26 +16,26 @@
 @Service
 public class PrivateDataRetrievalService {
 
-    private static final String OVERSEAS_ENTITY_URI_SECTION = "/overseas-entity/%s/entity-data";
+    private static final String OVERSEAS_ENTITY_URI_SECTION = "/company/%s/registered-email-address";
 
     @Autowired
     private ApiClientService apiClientService;
 
-    public OverseasEntityDataApi getOverseasEntityData(String requestId, String companyNumber)
+    public RegisteredEmailAddressJson getRegisteredEmailAddressData(String requestId, String companyNumber)
             throws ServiceException {
         var logDataMap = new DataMap.Builder().companyNumber(companyNumber).build();
         try {
             ApiLogger.infoContext(requestId, "Retrieving overseas entity data from database", logDataMap.getLogMap());
 
-            var overseasEntityDataApi = apiClientService
+            var registeredEmailAddress = apiClientService
                     .getInternalApiClient()
-                    .privateOverseasEntityDataHandler()
-                    .getOverseasEntityData(String.format(OVERSEAS_ENTITY_URI_SECTION, companyNumber))
+                    .privateCompanyResourceHandler()
+                    .getCompanyRegisteredEmailAddress(String.format(OVERSEAS_ENTITY_URI_SECTION, companyNumber))
                     .execute()
                     .getData();
 
             ApiLogger.infoContext(requestId, "Successfully retrieved overseas entity data from database", logDataMap.getLogMap());
```

Comment:
Log output can now be a bit more specific - "Successfully retrieved overseas entity email address from database".

Reply:
Updated all messages to be more generic while at the same time specifying "company registered email address"